geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeave.java

```diff
@@ -621,10 +621,12 @@ private void processLeaveRequest(LeaveRequestMessage incomingRequest) {
       Collection<InternalDistributedMember> suspectMembers =
           services.getHealthMonitor().getMembersFailingAvailabilityCheck();
       check.removeAll(suspectMembers);
-      logger.info(
-          "View with removed and left members removed is {}; removed members: {}; left members: {}; suspect members: {}",
-          check, removedMembers, leftMembers, suspectMembers);
+      logger.info("View with removed and left members removed is {}", check);
       if (check.getCoordinator().equals(localAddress)) {
+        for (InternalDistributedMember suspect : suspectMembers) {
```

Comment:
Can we add unit tests for the new changes; if its not there.

Reply:
yes, I'll do that